core/domain/email_manager_test.py

```diff
@@ -88,6 +90,198 @@ def test_sender_id_validation(self):
         # pylint: enable=protected-access
 
 
+class ExplorationMemebershipEmailTests(test_utils.GenericTestBase):
+    """Tests that sending exploration membership email works as expected."""
+
+    def setUp(self):
+        super(ExplorationMemebershipEmailTests, self).setUp()
+
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+        self.exploration = self.save_new_default_exploration(
+            'A', self.editor_id, 'Title')
+
+        self.email_subject_tempate = (
```

Comment:
Okay so golden email means raw string without any interpolation. So here we have 3 test for 3 different roles. So expected_email_body will be different for each of the tests, right? 
For e.g. expected_email_html_body for manager role will be something like below, correct ?
```
expected_email_html_body = (
            'Hi newuser,<br>'
            '<br>editor has granted you Manager rights to their '
            'learning exploration, '
            '<a href="http://www.oppia.org/A">"Title"</a>'
            ', on Oppia.org.<br>'
            'This allows you to:<br>'
            '<ul>'
            '<li>Change the exploration permissions</li>'
            '<li>Edit the exploration</li>'
            '<li>View and playtest the exploration</li>'
            '</ul><br>'
            'You can find exploration '
            '<a href="http://www.oppia.org/A">here</a>. Thanks!<br>'
            '<br>Best Wishes,<br>'
            'The Oppia Team<br>'
            '<br>You can change your email preferences via the '
            '<a href="https://www.example.com">Preferences</a> page.'
        )

```

Is that alright ?